docs/Ceph-Caching.md

```diff
@@ -7,20 +7,27 @@ First we need to have a working cluster.
 `minikube start --memory='10G' --cpus=8 --disk-size='15g' --driver=docker`
```

Comment:
Maybe have the part 2 be minikube only and hardcode the values that make sense both for the above (disk-size)

Reply:
implemented